README.md

```diff
@@ -25,6 +25,154 @@ Logback appender which posts logs to slack via incoming webhook.
+        <payload>
+            {
+              "channel": "#_channel",
+              "username": "username",
+              "icon_emoji": emoji,
+              "link_names": 1,
+              "attachments": [{
+                "title": level + " (" + hostname + ")",
+                "fallback": level + " (" + hostname + ")",
+                "color": color,
+                "fields": [{
+                  "title": "Hostname",
+                  "value": hostname,
+                  "short": true
+                }, {
+                  "title": "Time",
+                  "value": timestamp,
+                  "short": true
+                }, {
+                  "title": "Level",
+                  "value": level,
+                  "short": true
+                }, {
+                  "title": "Trigger",
+                  "value": message,
+                  "short": false
+                }]
+              }]
+            }
+        </payload>
+    </appender>
+
+    <appender name="ASYNC_SLACK" class="ch.qos.logback.classic.AsyncAppender">
+        <appender-ref ref="SLACK" />
+        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+            <level>INFO</level>
+        </filter>
+    </appender>
+
+    <root level="INFO">
+        <appender-ref ref="ASYNC_SLACK" />
+    </root>
+
+</configuration>
+```
+
+#### Appender options
+
+|Key|Required|Detail|
+|:----|:----|:----|
+|webhookUrl|Y|URL of incoming webhook.|
+|timeout|N|Timeout of posting to Slack in millisecounds. (Default 50,000 millisecounds)|
+|payload|Y|Payload written in Javascript to send to Slack.|
+
+#### Payload specification
+
+- Written in JavaScript
+- Available Buildin variables
+
+|Variable Name|Type|Description|
+|:----|:----|:----|
+|hostname|string|Hostname where the event occurred|
+|event|ILoggingEvent|The representation of logging events. see [API](https://logback.qos.ch/apidocs/ch/qos/logback/classic/spi/ILoggingEvent.html)|
+|level|string|Log level. eg. FATAL, ERROR, WARN, INFO, DEBUG, TRACE.|
+|timestamp|string|Time when the event occurred. eg. 2018-02-21T19:00:25.827+09:00|
+|message|string|The message of logging events.|
+|color|string|Color according to log level.|
+|emoji|string|Emoji according to log level.|
+
+##### Payload sample - Simple
+
+```javascript
+{
+  "channel": "#_channel",
+  "username": "username",
+  "icon_emoji": emoji,
+  "link_names": 1,
+  "attachments": [{
+    "title": level + " (" + hostname + ")",
+    "fallback": level + " (" + hostname + ")",
+    "color": color,
+    "fields": [{
+      "title": "Hostname",
+      "value": hostname,
+      "short": true
+    }, {
+      "title": "Time",
+      "value": timestamp,
+      "short": true
+    }, {
+      "title": "Level",
+      "value": level,
+      "short": true
+    }, {
+      "title": "Trigger",
+      "value": message,
+      "short": false
+    }]
+  }]
+}
+```
+
+##### Payload sample - Complex
+
+```javascript
+var env = 'dev', mention = '';
```

Comment:
I do not like to write some logic in payload. Why not using different logback.xml on each environment? If you really want to use conditional expression in a single xml file, you can use [if statement](https://logback.qos.ch/manual/configuration_ja.html#conditional) as well.